docs/source/layouts/layout.erb

```diff
@@ -11,6 +11,7 @@
 
     <%= stylesheet_link_tag :site %>
     <%= auto_stylesheet_link_tag %>
+    <%= javascript_include_tag config[:places_lib_url] %>
```

Comment:
Yes, easier to handle. Maybe we can move the JS code to `content_block` in the footer but I didn't investigate.